inductiva/resources/machine_cluster.py

```diff
@@ -52,7 +53,8 @@ def __init__(self,
     @classmethod
     def from_api_response(cls, resp: dict):
         machine_group = super().from_api_response(resp)
-        machine_group.num_machines = int(resp["num_vms"])
+        machine_group.num_machines = int(resp["max_vms"])
+        machine_group.__dict__["_current_machines"] = int(resp["num_vms"])
```

Comment:
I agree that this should be the way to go. For me it seems strange that I can do:

```
import inductiva

mg = inductiva.resources.ElasticMachineGroup(...2, 20)
# submit here 1e10 tasks
mg.current_machines_to_str() # ---> 0/20
```

Because the var is never updated from a backend response. However, if the backend still does not support this then I have no issues with this approach

Reply:
We don't have a get machine group by id. Once they implement that we can implement a method called update_current_machines().

Reply:
maybe the `_current_machines` attribute can make a backend request everytime to update its value.

Reply:
> maybe the `_current_machines` attribute can make a backend request everytime to update its value.

We don't have that request yet. We could get all active machine groups and filter them to get to our machine group, but that does not make sense. We should have an api call to get a machine group by id.